apps/omg_eth/lib/eth.ex

```diff
@@ -33,6 +33,29 @@ defmodule OMG.Eth do
   @type address :: <<_::160>>
   @type hash :: <<_::256>>
 
+  @doc """
+  Send transaction to be singed by a key managed by Ethereum node, geth or parity
+  """
+  @spec send_transaction(map()) :: {:ok, binary()} | {:error, any()}
+  def send_transaction(txmap, opts \\ []) do
+    case backend() do
+      :geth ->
+        Ethereumex.HttpClient.eth_send_transaction(txmap)
+
+      :parity ->
+        with {:ok, passphrase} <- get_signer_passphrase(txmap.from) do
+          opts = Keyword.merge([passphrase: passphrase], opts)
+          params = [txmap, Keyword.get(opts, :passphrase, "")]
+          Ethereumex.HttpClient.request("personal_sendTransaction", params, [])
+        end
+    end
+  end
+
+  def backend do
+    Application.get_env(:omg_eth, :eth_node, "geth")
```

Comment:
Minor: making missing config failing might be better than default to geth ?

Reply:
I concur, the default is defined with `{:system...}` in the config file already. `Application.fetch_env!` sounds ok to me here